lib/_tls_wrap.js

```diff
@@ -281,7 +281,7 @@ TLSSocket.prototype._wrapHandle = function(handle) {
                 tls.createSecureContext();
   res = tls_wrap.wrap(handle, context.context, options.isServer);
   res._parent = handle;
-  res._reading = handle._reading;
+  res.reading = handle.reading;
```

Comment:
@chrisdickinson can you think of a way to test this? I'd really like to get a release out in the next hour or so with this PR merged.